src/Twilio/Rest/Client.php

```diff
@@ -201,6 +172,25 @@ public function __construct(string $username = null, string $password = null, st
         }
     }
 
+    /**
+     * Determines argument value accounting for environment variables.
+     *
+     * @param string $arg The constructor argument
+     * @param string $envVar The environment variable name
+     * @return ?string Argument value
+     */
+    public function getArg(?string $arg = null, string $envVar = null): ?string {
```

Comment:
`arg` shouldn't default to null. This would imply you can call `getArg` with no args and expect something meaningful.